disclosure_backend/views.py

```diff
@@ -45,6 +45,8 @@ def location_view(request, fips_id):
         required: true
     response_serializer: LocationSerializer
     """
+    # XXX: no-op so flake8 doesn't consider this an unused import
+    LocationSerializer()
```

Comment:
I'm not familiar with flake8, but you might be able to ignore a single line with

    # noqa

or

    # nopep8

eslint/jshint/etc allow you to ignore specific rules, which is nice so you're only making exceptions for exactly what you want. I don't see an equivalent for they python tools.